### ISSUE TITLE
Notification: "command '' not found" when using gopls from HEAD

### ISSUE THREAD

**Issue by atombender** [microsoft/vscode-go#2920]
I've started getting this message when editing Go. It started appearing after upgrading to the latest version of gopls.

![Screen Shot 2019-11-27 at 13 16 24](https://user-images.githubusercontent.com/50314/69749511-a7c7d500-1118-11ea-8976-6d5e1f431db0.jpg)

Grepping reveals that it appears in `renderer2.log`:

```
1037:[2019-11-27 12:58:27.793] [renderer2] [error] command '' not found: Error: command '' not found
1038-    at u._tryExecuteCommand (file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3305:697)
1039-    at file:///Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:3305:578
```

Unfortunately, that's all there is. I can only assume it's related to the Go extension — but the stack trace doesn't give me anything more. It also happens sporadically, but only when editing Go. It only started occurring after upgrading gopls. I have not installed/upgraded any extensions recently, though I did upgrade to the newest VSCode point release.

Any ideas?

Latest VScode with extension version 0.11.9, gopls `c1736c0f`, [settings.json](https://gist.github.com/atombender/da8f3722b9504b0cd694e2065f0712d9).

**Comment by stamblerre** [comment]
Can you share the output of `gopls version`?

**Comment by atombender** [comment]
The commit hash is above. It's `HEAD`, as we speak.

**Comment by stamblerre** [comment]
Thanks for reporting! Will be fixed in [CL 209219](https://go-review.googlesource.com/c/tools/+/209219).

**Comment by ramya-rao-a** [comment]
@stamblerre This issue was present in the latest stable version of gopls or only in the master branch?

**Comment by atombender** [comment]
Didn't see that the commit had been pushed, sorry. Upgraded and will report back.

**Comment by stamblerre** [comment]
> @stamblerre This issue was present in the latest stable version of gopls or only in the master branch?

It was only on master.

**Comment by ramya-rao-a** [comment]
Thanks @atombender

**Comment by atombender** [comment]
While I no longer get the notification, I occasionally get the same log message. Ignorable?

**Comment by stamblerre** [comment]
I think there is another instance (in completion) where we should be sending a nil command instead of an empty one. I'll mail another CL.

**Comment by stamblerre** [comment]
@ramya-rao-a: These fixes will be part of `gopls/v0.3.0`, so I believe that this issue can be closed.
